libraries/AP_GPS/AP_GPS.h

```diff
@@ -260,6 +263,23 @@ class AP_GPS
         return ground_course_cd(primary_instance);
     }
 
+    // yaw in degrees if available
+    bool gps_yaw_deg(uint8_t instance, float &yaw_deg, float &accuracy_deg) const {
+        if (!have_gps_yaw(instance)) {
+            return false;
+        }
+        yaw_deg = state[instance].gps_yaw;
+        // None of the GPS backends can provide this yet, so we hard
+        // code a fixed value of 10 degrees, which seems like a
+        // reasonable guess. Once a backend can provide a proper
+        // estimate we can implement it
+        accuracy_deg = 10;
+        return true;
```

Comment:
This is quite big to be in the header. There are several other similar methods in the cpp file, this should be there.

Reply:
@OXINARF regarding the accuracy mentioned above in line 276 in libraries/AP_GPS/AP_GPS.h. Is this the expected accuracy of the GPS heading? Can I lower this value to give more weighting to the GPS heading?

Reply:
Yes, that's the accuracy. You can try lowering it, I have no idea if that's effective or not.